ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/source-item/source-item.tsx

```diff
@@ -116,11 +117,16 @@ export default hot(module)(({ id, sourceActions, available }: Props) => {
                       sourceAction.description
                     }`}
                     onClick={() => {
-                      lightboxInstance.model.updateTitle(sourceAction.title)
-                      lightboxInstance.model.open()
-                      lightboxInstance.showContent(
-                        new SourceAppView({ url: sourceAction.url })
-                      )
+                      if (sourceAction.id.startsWith('catalog.data.source.window')) {
+                        const windowFeatures = "location=yes,height=570,width=520,scrollbars=yes,status=yes"
```

Comment:
❓ what is the significance of the `height` and `width` here? Can we pull them up into variables which ascribe some meaning to them?

Reply:
Given that 99% of the time, this will be for authentication, it will open some auth window and I feel that having a smaller window is more appropriate.

Reply:
Added variable names for dimensions